scripts/build/rollup.parser.config.js

```diff
@@ -18,6 +18,15 @@ export default Object.assign(baseConfig, {
           include: "node_modules/typescript-eslint-parser/parser.js"
         })
       : {},
+    // In flow-parser 0.59.0 there's a dynamic require: `require(s8)` which not
+    // supported by rollup-plugin-commonjs, so we have to replace the variable
+    // by its value before bundling.
+    parser === "flow"
+      ? replace({
+          "require(s8)": 'require("fs")',
+          include: "node_modules/flow-parser/flow_parser.js"
+        })
+      : {},
```

Comment:
cc @mroch, not sure it was intentional but would be great to remove dynamic requires

Reply:
this is js_of_ocaml optimizing things... and comes from its internal library. hrm.

also, `s8` is the random name it happened to assign to that hoisted string so this is definitely unstable across releases. will have to think about this...

Reply:
Yeah, I'm aware this is a random name given in the ocaml -> JS build. But that shouldn't be a problem until we update flow again, and we don't do that often.

Reply:
But it's funny that the `require("constants")` remained like that and `require("fs")` got hoisted.